uploaded_code/uploaded_code.ino

```diff
@@ -1887,8 +1902,13 @@ void setField_MonthAndDate(int &field, int &month, int &date) {
     field = 10000 + month * 100 + date;
     Println(7, "From Month : " + String(month) + " & Date : " + String(date) +
                    " => field : " + String(field));
-  } else {
-    Println(7, "\tError in setField_MonthAndDate function!");
+  } else if (millis() > 20000) {
+    Println(7, "\tError in setField_MonthAndDate function!" +
+<<<<<<< HEAD
+                   String(millis() / 1000));
+=======
+                   Sting(millis() / 1000));
+>>>>>>> 158fb5b5088d8bedf878c206ae0b5d5e69a2e332
```

Comment:
same here

Reply:
my bad, actually i forgot 1 thing so I reset HEAD and and then make a commit which lead to this error.